liberica/prettier.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import("prettier").Config} */
+export default {
+    plugins: [ "prettier-plugin-tailwindcss" ],
+    trailingComma: "es5",
+    tabWidth: 4,
+    singleQuote: true,
+    bracketSameLine: true
```

Comment:
```suggestion
    tabWidth: 4,
```
I'd prefer the default options for `trailingComma`, `singleQuote` and `bracketSameLine`.